src/fonts.scss

```diff
@@ -0,0 +1 @@
+@import "../scss/fonts/72";
```

Comment:
Where is this used?

Reply:
We need it in fundamental react for the docs site at least.